projects/tuist/fixtures/app_with_spm_dependencies/StringifyMacro/Package.swift

```diff
@@ -1,4 +1,4 @@
-// swift-tools-version: 5.9
+// swift-tools-version: 5.8
```

Comment:
This was causing acceptance tests to fail because we don't use the new Xcode version on CI (yet)